test/performance/norway/travelSearch.csv

```diff
@@ -44,3 +44,45 @@ testCaseId,description,departure,arrival,window,origin,fromPlace,fromLat,fromLon
+219,Kvadraturen to Jordal,10:00,,,Kvadraturen,,59.9109,10.7423,Jordal,,59.90947,10.78350,scooter-rent,WALK|SCOOTER_RENT
+220,Kvadraturen to Lille Tøyen,10:00,,,Kvadraturen,,59.9109,10.7423,Lille Tøyen,,59.9204,10.7866,scooter-rent,WALK|SCOOTER_RENT
+221,Kvadraturen to Briskeby,10:00,,,Kvadraturen,,59.9109,10.7423,Briskeby,,59.9211,10.7167,scooter-rent,WALK|SCOOTER_RENT
+222,Kvadraturen to Bergskogen,10:00,,,Kvadraturen,,59.9109,10.7423,Bergskogen,,59.94770,10.74223,scooter-rent,WALK|SCOOTER_RENT
+223,Kvadraturen to Refstad,10:00,,,Kvadraturen,,59.9109,10.7423,Refstad,,59.9388,10.8012,scooter-rent,WALK|SCOOTER_RENT
+224,Kvadraturen to Eiksmarka,10:00,,,Kvadraturen,,59.9109,10.7423,Eiksmarka,,59.9463,10.6263,scooter-rent,WALK|SCOOTER_RENT
+225,Bergskogen to Jordal,10:00,,,Bergskogen,,59.94770,10.74223,Jordal,,59.90947,10.78350,scooter-rent,WALK|SCOOTER_RENT
+226,Bergskogen to Lille Tøyen,10:00,,,Bergskogen,,59.94770,10.74223,Lille Tøyen,,59.9204,10.7866,scooter-rent,WALK|SCOOTER_RENT
+227,Bergskogen to Briskeby,10:00,,,Bergskogen,,59.94770,10.74223,Briskeby,,59.9211,10.7167,scooter-rent,WALK|SCOOTER_RENT
+228,Bergskogen to Kvadraturen,10:00,,,Bergskogen,,59.94770,10.74223,Kvadraturen,,59.9109,10.7423,scooter-rent,WALK|SCOOTER_RENT
+229,Bergskogen to Refstad,10:00,,,Bergskogen,,59.94770,10.74223,Refstad,,59.9388,10.8012,scooter-rent,WALK|SCOOTER_RENT
+230,Bergskogen to Eiksmarka,10:00,,,Bergskogen,,59.94770,10.74223,Eiksmarka,,59.9463,10.6263,scooter-rent,WALK|SCOOTER_RENT
+231,Refstad to Jordal,10:00,,,Refstad,,59.9388,10.8012,Jordal,,59.90947,10.78350,scooter-rent,WALK|SCOOTER_RENT
+232,Refstad to Lille Tøyen,10:00,,,Refstad,,59.9388,10.8012,Lille Tøyen,,59.9204,10.7866,scooter-rent,WALK|SCOOTER_RENT
+233,Refstad to Briskeby,10:00,,,Refstad,,59.9388,10.8012,Briskeby,,59.9211,10.7167,scooter-rent,WALK|SCOOTER_RENT
+234,Refstad to Kvadraturen,10:00,,,Refstad,,59.9388,10.8012,Kvadraturen,,59.9109,10.7423,scooter-rent,WALK|SCOOTER_RENT
+235,Refstad to Bergskogen,10:00,,,Refstad,,59.9388,10.8012,Bergskogen,,59.94770,10.74223,scooter-rent,WALK|SCOOTER_RENT
+236,Refstad to Eiksmarka,10:00,,,Refstad,,59.9388,10.8012,Eiksmarka,,59.9463,10.6263,scooter-rent,WALK|SCOOTER_RENT
+237,Eiksmarka to Jordal,10:00,,,Eiksmarka,,59.9463,10.6263,Jordal,,59.90947,10.78350,scooter-rent,WALK|SCOOTER_RENT
+238,Eiksmarka to Lille Tøyen,10:00,,,Eiksmarka,,59.9463,10.6263,Lille Tøyen,,59.9204,10.7866,scooter-rent,WALK|SCOOTER_RENT
+239,Eiksmarka to Briskeby,10:00,,,Eiksmarka,,59.9463,10.6263,Briskeby,,59.9211,10.7167,scooter-rent,WALK|SCOOTER_RENT
+240,Eiksmarka to Kvadraturen,10:00,,,Eiksmarka,,59.9463,10.6263,Kvadraturen,,59.9109,10.7423,scooter-rent,WALK|SCOOTER_RENT
+241,Eiksmarka to Bergskogen,10:00,,,Eiksmarka,,59.9463,10.6263,Bergskogen,,59.94770,10.74223,scooter-rent,WALK|SCOOTER_RENT
+242,Eiksmarka to Refstad,10:00,,,Eiksmarka,,59.9463,10.6263,Refstad,,59.9388,10.8012,scooter-rent,WALK|SCOOTER_RENT
```

Comment:
would some combined cases with `WALK|TRANSIT|SCOOTER_RENT` be interesting to measure too?

Reply:
It would but for this case I'm interested in just the performance of the scooter rental since I am making changes to that.

Feel free to add them if you're interested. The folder now contains a script with which you can create test cases quickly.